packages/test-utilities/lib/create-span.ts

```diff
@@ -15,6 +16,15 @@ export function createSamplingProbability (rawProbability: number): SpanProbabil
   }
 }
 
+export function createSpan (name: string): Span {
```

Comment:
Can we remove the `name` param here as it's not being used?

Reply:
It will get used in the next PR to make assertions on spans a little easier